### PR TITLE
Add pomodoro_app in workshops

### PR DESCRIPTION
**Pull request by shyakachaste**
I added a pomodoroapp folder in workshops folder

### CODE REVIEW

**Review by grymmy — APPROVED** [review]
This looks fine to me but probably should have someone on clubs look at it

**Comment by grymmy** [comment]
@SerenityUX do you see any reason why this shouldn't land?

**Comment by shyakachaste** [comment]
okay thanks, let me wait for your final decision

**Pull request merged**
